oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -58,32 +58,47 @@
 
     // Linux defines a set of virtual file systems
     private static final List<String> PSEUDO_FS = Arrays.asList(//
```

Comment:
Can you please move this to the AbstractFileSystem object (and make it `protected`)?  The SolarisFileSystem and FreeBSDFileSystem need something similar.

Reply:
Ok, and you want those from Solaris and FreeBSD added into the same list?